modules/ROOT/pages/platform/security/single-sign-on.adoc

```diff
@@ -5,19 +5,110 @@
 label:AuraDB-Enterprise[]
 label:AuraDS-Enterprise[]
 
-Aura Enterprise supports Single Sign-On (SSO) at both the Console level and for accessing Workspace, Bloom and Browser clients directly at the instance level.
+== Introduction to SSO / OAuth
+
+Aura Enterprise supports Single Sign-On (SSO) at the organization, project and instance level.
+
+* Grant secure access to data and keep data safe.
+* At the same time, secure access for colleagues without being hammered down by security and data access teams.
+* If you have access to specific parts of the Aura Console, you can move between different parts of the Aura console without having to re-login.
+
+This documentation demonstrates how to easily set up Aura SSO for automatic access to data with Role Based Access Control (RBAC) so you can centrally control privileges (read/write/delete/etc) as well as data access (only access the data you are permitted to access).
+
+
+
+== Requirements
 
-[NOTE]
-====
 Accessing Aura with SSO requires:
  
-* Authorization Code Flow with PKCE.
+* Authorization Code Flow with PKCE (Proof Key for Code Exchange).
 * A publicly accessible Identity Provider (IdP) server.
-====
+
+== Configuring SSO 
+
+
+Customers can configure a combination of one or more of the following login methods:
+
+* Email/password
+* Okta
+* Microsoft Entra ID (previously Azure Active Directory)
+
+You can add Aura as a log in from a tile in an organization's Apps Dashboard.
+
+== Home Realm Discovery 
+
+Home Realm Discovery (HRD) is when a user enters their email on the login form, an Auth0 connection that is configured with a domain that matches the email domain will redirect the user to the Auth0 connection Idp.
+
+Users that have HRD enabled are redirected to a view where they can select from a list of their IdPs, if the customer has configured more than one IdP.
+
+
+
+= Information required in the Aura Console to set up SSO 
+
+To configure SSO from the Aura side, go to the *Aura Console > Settings > SSO Configuration.*
+
+The set-up form requires the following info:
+
+* SSO config ID - you decide a name for this
+* IdP type (e.g. Okta / Microsoft Entra ID)
+* Channel
+* Client ID 
+* Client Secret
+* Discovery URI
+* Issuer
+* Authorization Endpoint
+
+= Organization SSO
```

Comment:
```suggestion
== Organization SSO
```
If you want to brand new chapter, you have to create a new file. Only one top-level (=) header per file is allowed.